utm/conversion/9.411/ha_standalone.template

```diff
@@ -0,0 +1,1404 @@
+        },
+        "ExistingS3Bucket": {
+          "default": "S3 Bucket for UTM Synchronization and Backup (optional)"
+        },
+        "LicensePool": {
+          "default": "S3 Bucket Containing Licenses (optional)"
+        },
+        "Hostname": {
+          "default": "Hostname (required)"
+        },
+        "Email": {
+          "default": "Admin E-Mail Address (required)"
+        },
+        "AdminPassword": {
+          "default": "Admin Password (required)"
+        },
+        "Organization": {
+          "default": "Organization (required)"
+        },
+        "City": {
+          "default": "City (required)"
+        },
+        "Country": {
+          "default": "Country (required)"
+        },
+        "KeyName": {
+          "default": "SSH Key (required)"
+        },
+        "debugMode": {
+          "default": "Debug Mode"
+        }
+      }
+    }
+  },
+  "Mappings": {
+    "RegionMap": {
+      "ap-northeast-1": {
+        "Hourly": "ami-9ca7f6fb",
+        "BYOL": "ami-41a4f526",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "ap-northeast-2": {
+        "Hourly": "ami-59924137",
+        "BYOL": "ami-4a934024",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "ap-south-1": {
+        "Hourly": "ami-9d4b3bf2",
+        "BYOL": "ami-0148386e",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "ap-southeast-1": {
+        "Hourly": "ami-29cb784a",
+        "BYOL": "ami-bbc477d8",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "ap-southeast-2": {
+        "Hourly": "ami-72cfcd11",
+        "BYOL": "ami-21d2d042",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "ca-central-1": {
+        "Hourly": "ami-3e57ea5a",
+        "BYOL": "ami-0d58e569",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "eu-central-1": {
+        "Hourly": "ami-3b3ce854",
+        "BYOL": "ami-c93ce8a6",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "eu-west-1": {
+        "Hourly": "ami-cb446ead",
+        "BYOL": "ami-8a456fec",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "eu-west-2": {
+        "Hourly": "ami-42e9fc26",
+        "BYOL": "ami-1ce8fd78",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "sa-east-1": {
+        "Hourly": "ami-f373129f",
+        "BYOL": "ami-3071105c",
+        "ARN": "aws",
+        "HAInstanceType": "m3.large"
+      },
+      "us-east-1": {
+        "Hourly": "ami-0fc06119",
+        "BYOL": "ami-7cc1606a",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "us-east-2": {
+        "Hourly": "ami-f4684c91",
+        "BYOL": "ami-d96e4abc",
+        "ARN": "aws",
+        "HAInstanceType": "c4.large"
+      },
+      "us-gov-west-1": {
+        "Hourly": "ami-28932849",
```

Comment:
That's the 9.409 AMI id.
Should be 	
ami-437fc522
same for the other HA template